site/styles/_shopping.scss

```diff
@@ -13,11 +13,9 @@
 
     .s72-btn-close {
       background: transparent;
-      color: var(--body-color);
       display: block;
       height: 30px;
       line-height: 30px;
-      opacity: 1;
```

Comment:
Just confirming you've checked the shopping modal is good for light/dark themes?

Reply:
Sure did and double-checked just then also.

Note the above removed opacity/color now applied to shopping modal close via [this .extend](https://github.com/shift72/core-template/blob/develop/site/styles/_shift72.scss#L26-L28) and [this addition](https://github.com/shift72/core-template/pull/269/files#diff-37c2412e0517de1e975b3f3149c9fec0ee515c03503c64199b5e41ef94ec8103R56-R58)